test/spec/algoliasearch.helper/pages.js

```diff
@@ -1,72 +1,68 @@
 'use strict';
 
-var test = require('tape');
 var algoliasearchHelper = require('../../../index');
 
+var bind = require('lodash/bind');
+
 var fakeClient = {};
 
-test('setChange should change the current page', function(t) {
+test('setChange should change the current page', function() {
   var helper = algoliasearchHelper(fakeClient, null, null);
 
-  t.ok(helper.getCurrentPage() === 0, 'First page should be 0');
+  expect(helper.getCurrentPage() === 0).toBeTruthy();
```

Comment:
We can use `toBe(0)`.

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278628453